repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/glossary/CategoryPreProcessor.java

```diff
@@ -74,6 +95,8 @@ public void processAttributes(AtlasStruct entityStruct, EntityMutationContext co
                     entityStruct.getAttribute(QUALIFIED_NAME), operation);
         }
 
+        this.context = context;
```

Comment:
Can we pass it throughthe constructor?

Reply:
processAttributes is already defined with this input argument earlier so utilizing same way, in case we want to pass it via constructor all implementations of PreProcessor will need change